rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import resolve from 'rollup-plugin-node-resolve';
```

Comment:
Is there any chance `rollup.config.js` could be inside the `docs` because that's what it's for? Not sure if `entry` can live in a folder above it though...